region/norfair/crocomire.json

```diff
@@ -882,8 +1382,22 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    {"heatFrames": 100}
+                    {"heatFrames": 65}
                   ]
+                },
+                {
+                  "name": "Leave with Runway",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    {"heatFrames": 75}
```

Comment:
This should only include the heat frames to get to the start of the runway, so probably like 30 ish?

Reply:
Got it.  Cool how that works.